x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/location_stats.tsx

```diff
@@ -60,7 +60,10 @@ export function MobileLocationStats({
   const {
     path: { serviceName },
     query: { environment, offset, comparisonEnabled },
-  } = useAnyOfApmParams('/mobile-services/{serviceName}/overview');
+  } = useAnyOfApmParams(
+    '/mobile-services/{serviceName}/overview',
+    '/mobile-services/{serviceName}/transactions'
+  );
```

Comment:
What about removing it and receiving the `serviceName`, `environment`, `offset`, and `comparisonEnabled` as properties of this component?